oa4mp/serve.go

```diff
@@ -306,6 +306,28 @@ func ConfigureOA4MP() (launcher daemon.Launcher, err error) {
 		return
 	}
 
+	// Ensure the OA4MP storage directory exists and has correct permissions and ownership
+	OA4MPStoragePath := "/opt/scitokens-server/var/storage"
+	if _, err := os.Stat(OA4MPStoragePath); os.IsNotExist(err) {
```

Comment:
Skip the `Stat`; just go ahead and do a `os.Chmod` and catch the `os.IsNotExist` as you do currently.

If you do create the directory creation, do a follow-up `os.Chmod`.